extensions/objects/templates/numeric-expression-editor.component.spec.ts

```diff
@@ -65,7 +65,7 @@ describe('NumericExpressionEditor', () => {
     component = fixture.componentInstance;
     guppyInitializationService = TestBed.inject(GuppyInitializationService);
     deviceInfoService = TestBed.inject(DeviceInfoService);
-    window.Guppy = MockGuppy;
+    window.Guppy = MockGuppy as unknown as Guppy;
```

Comment:
Why `unknown`?

Reply:
Same explanation as above

Reply:
The explanation: https://github.com/oppia/oppia/pull/16073#discussion_r1047014911

Reply:
Added a comment and filed an issue